lib/server.js

```diff
@@ -0,0 +1,131 @@
+var Docker = require('dockerode');
+var debug = require('debug')('docker-exec-websocket-server:lib:server');
+var through = require('through');
+var fs = require('fs');
+var ws = require('ws');
+var slugid = require('slugid');
+var _ = require('lodash');
+var path = require('path');
+var assert = require('assert');
+
+var dockerSocket = '/var/run/docker.sock';
+var stats = fs.statSync(dockerSocket);
+if (!stats.isSocket()) {
+  throw new Error('Are you sure the docker is running?');
+}
+var docker = new Docker({socketPath: dockerSocket});
+
+var dbgstream;
+if (process.argv[2] === '--dump') {
+  dbgstream = fs.createWriteStream(path.join(__dirname, '/dump.log'));
+}
```

Comment:
How about we make this an option on `DockerExecWebsocketServer` it's constructor already takes an `option` dictionary...